### ISSUE TITLE
Neos.Fusion:Debug makes ui reload indefinitely

### ISSUE THREAD

**Issue by klfman** [neos/neos-ui#1664]
Searched the issues, did not find anything related.

### Description

when using Neos.Fusion:Debug to output debug information in the Neos backend the loading indicator (orange) in the top part of the screen cycles through indefinitely, suggesting, that the page is reloading all the time.
The desired debug information is rendered correctly, regardless.

### Steps to Reproduce

Just render this:

prototype(Vendor.Name:Foo) < prototype(Neos.Neos:ContentComponent) {
    renderer = Neos.Fusion:Debug {
        foo = 'bar'
    }
}

### Affected Versions

Neos: 3.3.6

(dev master, updated today, 27.2.2018)
neos/neos-ui dev-master f395830
neos/neos-ui-compiled  dev-master 806dad7

**Comment by hlubek** [comment]
I also noticed issues when adding content elements that output Debug information. It seems the JSON encoding of results will be broken by the Debug object.

**Comment by grebaldi** [comment]
possibly related with #1718

**Comment by mhsdesign** [comment]
Yes i think its okay for the UI to signal: Hey whats up i dindt expect these iframe contents.
But after lets say a while it should just stop the loading animation - its just killing me at the moment :P
I think we could check if we got something returned what dindt met our expectations we will just make the bar red above.
In case where its legit that the ui loads forever (a super large website) we should of course preserve the current behaviour,

**Comment by grebaldi** [comment]
Closed in favor of https://github.com/neos/neos-ui/issues/3477.
